<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PowerControl</class>
 <widget class="QWidget" name="PowerControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1804</width>
    <height>1322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_PowerControl">
     <item>
      <widget class="QGroupBox" name="groupBox_GPUPowerControl">
       <property name="title">
        <string>GPU Power Control </string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_PowerControl">
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutPowerControl_2">
            <item>
             <widget class="QLabel" name="label_GPUTempLimitPowerControl">
              <property name="toolTip">
               <string>When the graphics card reaches the temperature set in the thermal wall, it will automatically limit the frequency to control the temperature; this option
can be used to limit the power consumption of the GPU.</string>
              </property>
              <property name="text">
               <string>Temperature limit [°C]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_GPUTempLimitPowerControl">
              <property name="minimum">
               <number>75</number>
              </property>
              <property name="maximum">
               <number>87</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_GPUTempLimitPowerControl">
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_8">
              <item>
               <widget class="QLabel" name="label_GPUBoostClockPowerControl">
                <property name="toolTip">
                 <string>This is the maximum additional power that can be allocated to the GPU from the CPU based on CPU usage. The higher the value, the better the
performance of applications that use the GPU.</string>
                </property>
                <property name="text">
                 <string>CPU to GPU Dynamic Boost [W]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_GPUBoostClockPowerControl">
                <item>
                 <property name="text">
                  <string>0</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>15</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>20</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>25</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_9">
              <item>
               <widget class="QLabel" name="label_GPUTargetPowerPowerControl">
                <property name="toolTip">
                 <string>The additional amount of power consumption that can be allocated on top of the graphics card base power consumption; the higher the
consumption, the stronger the graphics card performance, and the greater heat dissipation capacity required.</string>
                </property>
                <property name="text">
                 <string>Configurable TGP  [W]</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_GPUTargetPowerPowerControl">
                <item>
                 <property name="text">
                  <string>80</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>100</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>120</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>140</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>150</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_PowerControl">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_10">
            <item>
             <widget class="QPushButton" name="pushButton_GPUPowerControlCancel">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_GPUPowerControlApply">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_CPUPowerControl">
       <property name="title">
        <string>CPU Power Control</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_PowerControl_5">
            <item>
             <widget class="QLabel" name="label_CPUTempLimitPowerControl">
              <property name="toolTip">
               <string>When the core reaches the temperature set in the thermal wall, it will automatically limit the frequency to control the temperature; this option can be
used to limit the power consumption of the CPU.</string>
              </property>
              <property name="text">
               <string>Temperature limit [°C]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_CPUTempLimitPowerControl">
              <property name="minimum">
               <number>85</number>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_CPUTempLimitPowerControl">
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_PowerControl_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_PowerControl_4">
            <item>
             <widget class="QLabel" name="label_CrossLPowerLimitPowerControl">
              <property name="toolTip">
               <string>The setting value of maximum CPU power consumption when CPU and GPU are simultaneously fully loaded; the higher the value, the stronger the
CPU performance, and the greater heat dissipation capacity required.</string>
              </property>
              <property name="text">
               <string>Long Term Power Limit (Cross Loading)  [W]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_CrossLPowerLimitPowerControl">
              <property name="minimum">
               <number>20</number>
              </property>
              <property name="maximum">
               <number>110</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_CrossLPowerLimitPowerControl">
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayoutPowerControl">
            <item>
             <widget class="QLabel" name="label_STPowerLimitPowerControl">
              <property name="toolTip">
               <string>The peak power consumption that can be reached within a short time with pure CPU or AMD Smart Shift System load. The higher the value, the higher
the instantaneous performance of the CPU or System.</string>
              </property>
              <property name="text">
               <string>Platform Short Term Power Limit [W]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_STPowerLimitPowerControl">
              <property name="minimum">
               <number>55</number>
              </property>
              <property name="maximum">
               <number>190</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_STPowerLimitPowerControl">
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_PowerControl_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_PowerControl_3">
            <item>
             <widget class="QLabel" name="label_LTPowerLimitPowerControl">
              <property name="toolTip">
               <string>The setting value of maximum power consumption during continuous operation with pure CPU or AMD Smart Shift System load. The higher the value,
the greater the performance of the CPU or System and the greater heat dissipation capacity required.</string>
              </property>
              <property name="text">
               <string>Platform Long Term Power Limit [W] </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_LTPowerLimitPowerControl">
              <property name="minimum">
               <number>30</number>
              </property>
              <property name="maximum">
               <number>140</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_LTPowerLimitPowerControl">
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_PowerControl_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_PowerControl_2">
            <item>
             <widget class="QLabel" name="label_PowerTargetInACPowerControl">
              <property name="toolTip">
               <string>The point at which the CPU triggers dynamic power consumption adjustment for the graphics card. Use this point as the setting value at which CPU
and GPU dynamic power allocation begins.</string>
              </property>
              <property name="text">
               <string>Total Processor Power Target In AC [W]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_PowerTargetInACPowerControl">
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>110</number>
              </property>
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNumber_PowerTargetInACPowerControl">
            <property name="segmentStyle">
             <enum>QLCDNumber::SegmentStyle::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayoutPowerControl">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_2">
            <item>
             <widget class="QLabel" name="label_PL1TauPowerControl">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="toolTip">
               <string>PL1 power consumption start time.</string>
              </property>
              <property name="text">
               <string>PL1 Tau [s]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_PL1TauPowerControl">
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::SizeAdjustPolicy::AdjustToContents</enum>
              </property>
              <property name="placeholderText">
               <string/>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <item>
               <property name="text">
                <string>20</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>24</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>28</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>32</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>40</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>48</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>56</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>64</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>80</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>96</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>112</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>128</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>160</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutPowerControl_3">
            <item>
             <widget class="QLabel" name="label_CPUBoostPowerControl">
              <property name="toolTip">
               <string>This is the maximum additional power that can be allocated to the CPU from the GPU based on CPU usage. The higher the value, the better the
performance of applications that use the CPU.</string>
              </property>
              <property name="text">
               <string>GPU to CPU Dynamic Boost [W]</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox_CPUBoostPowerControl">
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>10</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>15</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacerPowerControl">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_PowerControl_5">
            <item>
             <widget class="QPushButton" name="pushButton_CPUPwoerControlCancel">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_CPUPowerControlApply">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_PowerControl">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
